### PR TITLE
feat(FTL-5593): add origin support

### PR DESCRIPTION
**Pull request by dhuan-affinidi**
# What changed

- `@affinidi/internal-api-clients`:
  - Add `origin` support so that when making requests to Registry, it is passed;

- `@affinidi/wallet-core-sdk`:
  - Update `SdkOptions` adding new optional field `origin`;
  - Pass `origin` to `@affinidi/internal-api-clients`;

### CODE REVIEW

File: common-libs/internal-api-clients/src/spec/_registry.ts [hunk 1]
```diff
@@ -60,6 +60,9 @@ export default {
           },
           "publicKeyBase58": {
             "type": "string"
+          },
```
Comment (anton-iskryzhytskyi) [inline common-libs/internal-api-clients/src/spec/_registry.ts:63]: I do not see this property in [CreateAnchorTransactionInput](https://gitlab.com/affinidi/foundational/affinity-registry/-/blob/main/src/did/did.dto.ts#L58), only in AnchorDidInput, which is already presented [here](https://github.com/affinityproject/affinidi-core-sdk/pull/232/files#diff-0874f74a245c4a669915ff56a14e08b4168ee64eaf80d50203ccf73a1af2c055R138).

File: sdk/core/src/services/anchoringHandler.ts [hunk 2]
```diff
@@ -37,7 +44,7 @@ const computePreparedJoloParams = async ({ registry, keysService, nonce, additio
 
   const {
     body: { digestHex },
-  } = await registry.createAnchorTransaction({ nonce, did, didDocumentAddress })
+  } = await registry.createAnchorTransaction({ nonce, did, didDocumentAddress, origin })
```
Comment (anton-iskryzhytskyi) [inline sdk/core/src/services/anchoringHandler.ts:47]: there are no such property in registry for  [createAnchorTransaction](https://gitlab.com/affinidi/foundational/affinity-registry/-/blob/main/src/did/did.dto.ts#L58)

File: sdk/core/src/services/anchoringHandler.ts [hunk 3]
```diff
@@ -51,14 +58,14 @@ const computePreparedElemParams = async ({ did }: AnchoringParams) => {
...
-const computePreparedPolygonParams = async ({ did, keysService, didMethod, registry }: AnchoringParams) => {
+const computePreparedPolygonParams = async ({ did, keysService, didMethod, registry, origin }: AnchoringParams) => {
   const didService = new PolygonDidDocumentService(new LocalKeyVault(keysService), {
     isTestnet: didMethod === 'polygon:testnet',
   })
   const publicKeyBase58 = didService.getMyPubKeyBase58()
   const {
     body: { digestHex },
-  } = await registry.createAnchorTransaction({ did, publicKeyBase58 })
+  } = await registry.createAnchorTransaction({ did, publicKeyBase58, origin })
```
Comment (anton-iskryzhytskyi) [inline sdk/core/src/services/anchoringHandler.ts:68]: ^

**Review by anton-iskryzhytskyi — CHANGES_REQUESTED** [review]
@dhuan-affinidi Could you please address my comments?

**Comment by dhuan-affinidi** [comment]
Anton, thanks I'll address this in a separate PR in the Registry service.

**Review by chebotiuk — APPROVED** [review]

File: sdk/core/src/services/anchoringHandler.ts (hunk 2 above)
Comment (chebotiuk) [reply sdk/core/src/services/anchoringHandler.ts:47]: @anton-iskryzhytskyi Do we need to add this field to registry then?

Comment (anton-iskryzhytskyi) [reply sdk/core/src/services/anchoringHandler.ts:47]: yes, did is created when we anchoring-did, not a transaction, it is old naming, and from my point of view it is not accurate.
https://affinity-registry.prod.affinity-project.org/api/redoc#tag/DID/operation/AnchorDid only when you need to send "origin"

File: sdk/core/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
+# release 6.6.0 (2022-09-21)
+* Added `origin` to `AnchorDidInput`
+# release 6.5.2 (2022-09-21)
```
Comment (chebotiuk) [inline sdk/core/CHANGELOG.md:3]: Added this, has been missed for https://github.com/affinityproject/affinidi-core-sdk/pull/242

**Review by anton-iskryzhytskyi — APPROVED** [review]

**Review by oleksiipiliugin — APPROVED** [review]

**Pull request merged**